src/phasorpy/phasor.py

```diff
@@ -818,15 +852,167 @@ def phasor_from_polar(
     (array([1, 0.5, 0.0]), array([0, 0.5, 1]))
 
     """
-    phase = numpy.asarray(phase)
-    modulation = numpy.asarray(modulation)
-    if phase.shape != modulation.shape:
-        raise ValueError(f'{phase.shape=} != {modulation.shape=}')
-    real = numpy.cos(phase)
-    real *= modulation
-    imag = numpy.sin(phase)
-    imag *= modulation
-    return real, imag
+    return _phasor_from_polar(phase, modulation, **kwargs)
+
+
+def phasor_to_apparent_lifetime(
```

Comment:
Sure, `lifetime_from_phasor` is a nicer name. However, it is misleading, since calculating general lifetime distributions from phasor coordinates (as that name would suggest) requires multiple harmonics/frequencies and fitting (better global fitting), which is out of the scope of this library at least for now.
I actually added a note for this in the module description but decided to move it into a "lifetime" tutorial (not included in this PR). Should I add that note back here?

Reply:
You are right, I think the tutorial es enough and since there is no general lifetime calculation for now it will not generate confusion in that aspect. So I think we should keep it as it is.